apps/web/src/components/home/GetUser.tsx

```diff
@@ -5,7 +5,7 @@ import { useAuth } from "@/contexts/Auth.hook";
 
 export const GetUser: React.FC = () => {
   const { user } = useAuth();
-  const name  = user !== null ? "Użytkowiku" : user?.nick.split(" ")[0]
+  const name  = user !== null ? "Użytkowniku!" : user?.nick.split(" ")[0]
```

Comment:
tej literówki u siebie nie poprawiłem, ale naprawiłem tego checka, aktualnie jako zalogowany użytkownik dalej będzie widoczny napis "Użytkowniku", zerknij jak to u mnie wygląda na branchu